### ISSUE TITLE
Quick search not working with regular user

### ISSUE THREAD

**Issue by BastiTee** [gcmd/gnome-commander#9]
Quick search seems not to work on a basic ubuntu machine. This behaviour has been described [here](https://bugs.launchpad.net/ubuntu/+source/gnome-commander/+bug/1045534?comments=all) but I am not sure how this repo and the launchpad are related.

I installed a mint system 
```DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=16.04
DISTRIB_CODENAME=xenial
DISTRIB_DESCRIPTION="Ubuntu 16.04.1 LTS"
```
and compiled from scratch the current [HEAD 066ec71c823bf325b7b958190d36ccbf1926d03f](https://github.com/gcmd/gnome-commander/commit/066ec71c823bf325b7b958190d36ccbf1926d03f). 

The observation: 

> The "Quick Search" function by typing in letters or by using the menu entry is not working. The first typed in digit opens the small command line window at the bottom as expected, all further typed digits are not accepted , the command line window freezes. Escape out of "Quick Search"'s command line is only possible by using left-mouse-click, ESC-key is not working.

This does not happen when I start gnome-commander as root.

Thanks in advance and keep up the good work!

**Comment by BastiTee** [comment]
Oh well... https://bugzilla.gnome.org/show_bug.cgi?id=686121

**Comment by turboscholz** [comment]
Hi!

Thank you for your notice. Yes, the bug was reported some time ago. But don't know when I will find the time to fix it. Currently my computer is broken :-(

**Comment by turboscholz** [comment]
Hi! Did you maybe tried the latest version of Gnome-Commander? I implemented a new option to search via quick search by just typing letters (without combination of additional keys). I did this in Xfce4 under Fedora 25. Here, all three possible options of how to use quick search work without the error you described above. So it might be related to Ubuntu or the desktop environment/settings you are using.
